extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -206,6 +206,26 @@ $(function () {
 			return result;
 		}
 
+		function trackSave() {
+			Wikia.Tracker.track({
+				action: Wikia.Tracker.ACTIONS.CLICK,
+				category: 'special-curated-content',
+				label: 'save',
+				trackingMethod: 'analytics'
+			});
+		}
+
+		function trackSaveError() {
+			Wikia.Tracker.track({
+				action: Wikia.Tracker.ACTIONS.CLICK,
+				category: 'special-curated-content',
+				label: 'save-error',
+				trackingMethod: 'analytics'
+			});
+		}
```

Comment:
you can create only one function and pass different label depending on the error